docs/navigation.md

```diff
@@ -45,6 +46,8 @@ Next, install the required peer dependencies. You need to run different commands
 
 > Note: You might get warnings related to peer dependencies after installation. They are usually caused by incorrect version ranges specified in some packages. You can safely ignore most warnings as long as your app builds.
 
+<M1Cocoapods />
```

Comment:
Commenting here but valid for the whole PR: instead of copying over the block, I'd rather add a smaller info box here, with a anchor link to https://reactnative.dev/docs/environment-setup#cocoapods where instructions are available.

Reply:
Ouch I published without seeing @Simek 's comment first, my bad. That approach also works 👍

Reply:
Do you mean something like: 
_"In case of issue with Cocoapods on a Mac M1, look at [how to set them up properly](https://reactnative.dev/docs/environment-setup#cocoapods)."_ 
?

Reply:
Yes exactly